scripts/dbaasp/dbaasp.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+N=8
```

Comment:
the script might get blocked if used with <8 cores, read how many cores is available (-1) or let the user decide